### PR TITLE
sqlflow integrate with zeppelin (EnvironmentSpecificSQLFlowClient.java)

### PR DESCRIPTION
**Pull request by Klay1124**
fix #3

Create a class of sqlflow client. According to the parameters obtained by the foreground: database address, user name, password and other information to build a user object.

In this code document, only MySQL is used for the time being. Other SQL engines include Oracle, Hive, SparkSQL, Flink, etc. developers can expand and improve them according to their actual needs.

In the code, `submitter = "mlp"` and `userId = "314"` are equivalent to the user's user name information and ID number. This is based on the user information of login Zeppelin. Different login users have different tags. Sqlflow can identify which user is using it. A fixed value is given in the case. Developers can set the values of `submitter` and `userId` as variables, and get the corresponding parameters according to the login user information of Zeppelin.

### CODE REVIEW

File: EnvironmentSpecificSQLFlowClient.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+		// Get the configuration parameters from the page side
+		String serverAddr = parameters.get("sqlflow.serverAddr");
+		String submitter = "mlp";
+		String username = parameters.get("mysql.username");
+		String password = parameters.get("mysql.password");
+		String mysqlAddr = parameters.get("mysql.serverAddr");
+		String databaseName = parameters.get("mysql.databaseName");
+		String dataSource = "mysql://" + username + ":" + password + "(" + mysqlAddr
+				+ ")/" + databaseName + "?maxAllowedPacket=0";
+		String userId = "314";
```
Comment (weiguoz) [inline EnvironmentSpecificSQLFlowClient.java]: What is the '314' means?

Comment (Klay1124) [reply EnvironmentSpecificSQLFlowClient.java]: "314" in this case is used to identify users in the background, and it gives a fixed value. Developers can set the value of **userId** as a variable and get the corresponding parameters according to the login user information of Zeppelin. Different login users have different **userId**, so sqlflow can identify which user is using it.

Comment (weiguoz) [reply EnvironmentSpecificSQLFlowClient.java]: So the `userId` should be a variable instead of a fixed one.

Comment (Klay1124) [reply EnvironmentSpecificSQLFlowClient.java]: Yes it is. The value given in this case is a fixed value, which can actually be assigned to `userId` by getting the user name of the zeppelin logged in user.

Comment (weiguoz) [reply EnvironmentSpecificSQLFlowClient.java]: How？

**Review by weiguoz — APPROVED** [review]
LGTM

**Pull request merged**